src/conv/heur/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM onnxmlirczar/onnx-mlir-build:x86
```

Comment:
This should use a digest to ensure this image tag doesn't change `FROM onnxmlirczar/onnx-mlir-build:x86@sha256:c8fdeb4f771d13398516e653ecb7ecc1d439c7ac380fc41927f1a4733c2ae092`